app/scripts/react-directives/users/pageStore.js

```
@@ -0,0 +1,235 @@
+  }
+
+  async processFetchError(fetchResponse) {
+    switch (fetchResponse.status) {
+      case 403:
+        this.pageWrapperStore.setError(i18n.t('users.errors.forbidden'));
+        break;
+      case 404:
+        this.pageWrapperStore.setError(i18n.t('users.errors.old-backend'));
+        break;
+      default:
+        const text = await fetchResponse.text();
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)